src/index.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+const logger = require('js-logger');
+const config = require ('./config.js');
+const sync = require ('./sync.js');
+const common = require ('./common.js');
+
+logger.useDefaults();
+
+cfg = config.readconfig();
+
+synctron = new sync.Sync(cfg, logger);
```

Comment:
Always use var declaration types

```suggestion
const cfg = config.readconfig();

const synctron = new sync.Sync(cfg, logger);
```